internal/pkg/validator/custom.go

```diff
@@ -187,3 +189,13 @@ func (v *wrapper) registerCustomRules() {
 		},
 	)
 }
+
+func (v *wrapper) registerCustomTypes() {
+	// Convert value to string using String method
+	v.validator.RegisterCustomTypeFunc(func(field reflect.Value) interface{} {
+		if v, ok := field.Interface().(fmt.Stringer); ok {
+			return v.String()
+		}
+		return field
+	}, model.SemVersion{})
```

Comment:
There are some fields with the type `model.SemVersion`, the value is no more automatically converted to string, so we need to register the callback.